app/controllers/events_controller.rb

```diff
@@ -1,3 +1,34 @@
 class EventsController  < ApplicationController
 
+	def new
+		@event = Event.new
+	end
+
+
+
+	def create
+ 		@event = Event.new(event_params)
+ 
+  		@event.save
+  		redirect_to @event
+	end
+
+
+	def destroy
+		@event = Event.find(params[:id])
+		@event.destroy
+
+		redirect_to events_path
+	end
+
+
+	private
+
+	def event_params
+		params.require(:event).permit(:description)      #richtig?
```

Comment:
Why only permitting `:description`? Shouldn't `title`, `start_time` and `end_time` be there, too?

Reply:
Will be fixed in this issue: https://github.com/ruby-stars/community-calendar/issues/24